src/selectors/project.js

```diff
@@ -0,0 +1,42 @@
+import { createSelector } from 'reselect'
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'reselect'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->